### ISSUE TITLE
[Bug][Accessibility] Tel: protocol is not supported by some desktop browsers and tablets.

### ISSUE THREAD

**Issue by gepifanio** [unimelb/pattern-lib#1361]
### Problem:

Some of the "href" links use "tel" protocol, however, it is not supported by some desktop browsers.

eg:

On desktops, laptops, and devices without a mobile SIM or Skype an error message is usually displayed:

Chrome on Windows 7 does nothing when the link is clicked
Firefox on Windows 7 displays the message 'The address wasn't understood'
IE11 on Windows 7 displays the message 'The webpage cannot be displayed'
Chrome on Windows 10 shows a 'Pick an app' window when the link is clicked
Edge on Windows 10 prompts you to use Skype or search for an app in the Windows Store
Firefox on Windows 10 shows a 'Pick an app' window when the link is clicked
IE11 on Windows 10 prompts you to use Skype or search for an app in the Windows Store
Chrome on macOS 10.13 launches Facetime which cannot make phone calls unless it's paired to an iPhone
Firefox on macOS 10.13 launches Facetime which cannot make phone calls unless it's paired to an iPhone
Safari on macOS 10.13 launches Facetime which cannot make phone calls unless it's paired to an iPhone
Safari on iPad with iOS 10.2 gives an option of adding a contact or sending a message
Chrome on Android shows a 'Add to Contacts' window when the link is clicked

### Possible solution:

- use JS handle the href and modify it for desktop / mobile use cases.

**Comment by dyarmachok** [comment]
Question regarding behaviour on desktop - Which way should I handle it ? To exclude the possibilty to click on phone number and make a call via Skype etc and leave it for mobile devices as it is ? 
We can do this for desktop by:
1. Changes in HTML CSS 
2. Building JS

or any other solution you expect, please let me know.

**Comment by jaredhenderson** [comment]
Hey @gepifanio , having this `tel` functionality is not important, and most phones are smart enough to detect phone numbers automatically anyway. We should avoid using JS to check/change this type of thing as it's really unnecessary and just adds complexity.

[Today's footer design](https://pattern-lib-unimelb.netlify.app/?selectedKind=Footer&selectedStory=Alt&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel) for the homepage doesn't use anything special for those numbers.

Let's just remove the `tel` links and leave them as text.
